test/data/lightning/test_datamodule.py

```diff
@@ -52,26 +55,32 @@ def forward(self, x, batch):
         x = self.lin2(x)
         return x
 
-    def training_step(self, data, batch_idx):
+    def training_step(self, data: Data, batch_idx: int):
```

Comment:
```suggestion
    def training_step(self, data: Data):
```
Since we don't use `batch_idx`, can we drop it?

Reply:
Thanks. It seems that even though `batch_idx` is not used, PyTorch Lightning still need that to index the batch of data (https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#training-step). I try to remove that but then it will raise an error
```
TypeError: test_step() takes 2 positional arguments but 3 were given
```